rust-runtime/aws-smithy-http-server/src/routing/into_make_lambda_service.rs

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use http::uri;
+use lambda_http::{Request, RequestExt};
+use std::{
+    fmt::Debug,
+    task::{Context, Poll},
+};
+use tower::Service;
+
+/// A [`MakeService`] that produces AWS Lambda compliant services.
```

Comment:
I don't think this qualifies as a [MakeService](https://docs.rs/tower/latest/tower/make/trait.MakeService.html#implementors). A `MakeService` is roughly a `Service` which given a request produces a `Service` as it's `Response`.

`hyper` requires you provide a ~`MakeService` to `Builder::serve` however `http_lambda::run` requires you provide a ~`Service<http::Request, Response = http::Response>`.

Reply:
`struct` name has changed but looks like the docs still refer to it as a `MakeService`.

Reply:
Thanks, fixed in: https://github.com/awslabs/smithy-rs/pull/1551/commits/32762e5044ed17fb721726b5e0b3ba8d1e37053d